homeassistant/components/apcupsd/sensor.py

```diff
@@ -524,3 +525,42 @@ def _update_attrs(self) -> None:
         self._attr_native_value, inferred_unit = infer_unit(self.coordinator.data[key])
         if not self.native_unit_of_measurement:
             self._attr_native_unit_of_measurement = inferred_unit
+
+    async def async_added_to_hass(self) -> None:
+        """Handle when entity is added to Home Assistant.
+
+        If this is a deprecated sensor entity, create a repair issue to guide
+        the user to disable it.
+        """
+        await super().async_added_to_hass()
+
+        if not self.enabled:
+            return
```

Comment:
I'm not super sure if we should also check if the entity is used in automations / scripts or if this is good enough?